syft/frameworks/torch/tensors/interpreters/large_precision.py

```diff
@@ -0,0 +1,194 @@
+import numpy as np
+import torch
+
+from syft.frameworks.torch.overload_torch import overloaded
+from syft.frameworks.torch.tensors.interpreters import AbstractTensor
+
+
+class LargePrecisionTensor(AbstractTensor):
+    """LargePrecisionTensor allows handling of numbers bigger than LongTensor
+
+    Some systems using Syft require larger types than those supported natively. This tensor type supports arbitrarily
+    large values by packing them in smaller ones.
+    These smaller values are of type `internal_type`. The large value is defined by `precision_fractional`
+    """
+
+    def __init__(
+        self,
+        owner=None,
+        id=None,
+        tags=None,
+        description=None,
+        internal_type=torch.int32,
+        precision_fractional=0,
+        verbose=False,
+    ):
```

Comment:
With respect to my comment below maybe we should consider add a `base` attribute just like in the precision tensor